nautobot/ipam/forms.py

```diff
@@ -503,6 +438,11 @@ class PrefixFilterForm(
         null_option="Global",
     )
     present_in_vrf_id = DynamicModelChoiceField(queryset=VRF.objects.all(), required=False, label="Present in VRF")
+    type = forms.MultipleChoiceField(
+        required=False,
+        choices=PrefixTypeChoices,
+        widget=StaticSelect2Multiple(),
+    )
```

Comment:
Should we also add an RIR field to the FilterForm? (I don't know whether we're going to get rid of FilterForms entirely in 2.0 at this point?)

Reply:
It's showing up automatically in the dynamic groups filter form and the advanced filter form. Is there anywhere else that would benefit from adding it to the simple filter form?

Reply:
Just the "non-advanced" filter tab, AFAIK.